app/api/services/briefs.py

```diff
@@ -35,11 +35,13 @@ def get_user_briefs(self, current_user_id):
         """Returns summary of a user's briefs with the total number of sellers that applied."""
         results = (db.session.query(Brief.id, Brief.data['title'].astext.label('name'),
                                     Brief.closed_at, Brief.status, func.count(BriefResponse.id).label('applications'),
-                                    Framework.slug.label('framework'), Lot.slug.label('lot'))
+                                    Framework.slug.label('framework'), Lot.slug.label('lot'),
+                                    WorkOrder.id.label('work_order'))
```

Comment:
would this also change the swagger on the corresponding view?

Reply:
Yes.  I've updated the appropriate docstring.